auth/providers/azure/options.go

```diff
@@ -17,18 +17,21 @@ type Options struct {
 	ClientID     string
 	ClientSecret string
 	TenantID     string
+	UseGroupUID  bool
 }
 
 func NewOptions() Options {
 	return Options{
 		ClientSecret: os.Getenv("AZURE_CLIENT_SECRET"),
+		UseGroupUID: true,
```

Comment:
I have changed it to default `true` , since that is the safer option.

Reply:
I left it `false` for backward compatibility reasons so that upgrading existing Guard deployments are not impacted.